### PR TITLE
Updated and cleaned generic styles

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: src/app/public/src/styles/_variables.scss [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 $stache-grid-columns: 12 !default;
 $stache-grid-padding: $sky-margin-plus-half !default;
-$stache-nav-item-padding: $sky-nav-item-padding !default;
 
 $stache-padding: $sky-padding !default;
 $stache-padding-double: $sky-padding-double !default;
 
-$stache-border-radius: 5px;
-$stache-border-radius-sm: 3px;
+$stache-border-radius: 3px;
```
Comment (Blackbaud-BrandonJones) [inline src/app/public/src/styles/_variables.scss]: Is there a reason we are going with out own border size here instead of the 4px sky uses?

Comment (Blackbaud-SteveBrush) [inline src/app/public/src/styles/_variables.scss]: Where are you seeing the SKY border-radius variable?

File: src/app/demos/default-styles/index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  <h5>
+    Heading 5
+  </h5>
+
+  <h6>
+    Heading 6
+  </h6>
+
+  <p>
+    Here, then, was this grey-headed, ungodly old man, chasing with curses a Job's whale round the world, at the head of a crew, too, chiefly made up of mongrel renegades, and castaways, and cannibals—morally enfeebled also, by the incompetence of mere unaided virtue or right-mindedness in Starbuck, the invunerable jollity of indifference and recklessness in Stubb, and the pervading mediocrity in Flask. Such a crew, so officered, seemed specially picked and packed by some infernal fatality to help him to his monomaniac revenge. How it was that they so aboundingly responded to the old man's ire—by what evil magic their
```
Comment (blackbaud-brandonhare) [inline src/app/demos/default-styles/index.html]: This feels AWFULLY whimsical @Blackbaud-SteveBrush. Awfully whimsical indeed.

File: src/app/demos/default-styles/index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        Table Data
+      </td>
+    </tr>
+  </table>
+
+  <ul>
+    <li>Unordered List</li>
+    <li>Passepartout was enraged beyond expression by the unpropitious weather</li>
+    <li>Had the hour of adversity come? Passepartout was as much excited as if the twenty thousand pounds were to come from his own pocket</li>
+    <li>Passepartout was enraged beyond expression by the unpropitious weather. Everything had gone so well till now! Earth and sea had seemed to be at his master's service; steamers and railways obeyed him; wind and steam united to speed his journey. Had the hour of adversity come? Passepartout was as much excited as if the twenty thousand pounds were to come from his own pocket. The storm exasperated him, the gale made him furious, and he longed to lash the obstinate sea into obedience. Poor fellow! Fix carefully concealed from him his own satisfaction, for, had he betrayed it, Passepartout could scarcely have restrained himself from personal violence.</li>
```
Comment (blackbaud-brandonhare) [inline src/app/demos/default-styles/index.html]: What sort of playfulness is this! Get outta here. This is a professional setting sir!

File: src/app/demos/default-styles/index.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  </ul>
+
+  <ol>
+    <li>Ordered List</li>
+    <li>Passepartout was enraged beyond expression by the unpropitious weather</li>
+    <li>Had the hour of adversity come? Passepartout was as much excited as if the twenty thousand pounds were to come from his own pocket</li>
+    <li>Passepartout was enraged beyond expression by the unpropitious weather. Everything had gone so well till now! Earth and sea had seemed to be at his master's service; steamers and railways obeyed him; wind and steam united to speed his journey. Had the hour of adversity come? Passepartout was as much excited as if the twenty thousand pounds were to come from his own pocket. The storm exasperated him, the gale made him furious, and he longed to lash the obstinate sea into obedience. Poor fellow! Fix carefully concealed from him his own satisfaction, for, had he betrayed it, Passepartout could scarcely have restrained himself from personal violence.</li>
```
Comment (blackbaud-brandonhare) [inline src/app/demos/default-styles/index.html]: Lorem ipsum or BUST!

File: src/app/public/src/styles/_variables.scss (hunk 1 above)
Comment (Blackbaud-BrandonJones) [inline src/app/public/src/styles/_variables.scss]: there isn't a variable. But they do have borders documented in their style guide... which... seems to have been updated in the past couple days...?

https://developer.blackbaud.com/skyux2/design/borders

so never mind lol

**Pull request merged**